src/graphics/renderer.rs

```diff
@@ -18,7 +18,7 @@ struct RenderSection {
 }
 
 pub struct Renderer {
-    size: Vector2D<i32>,
+    pub size: Vector2D<i32>,
```

Comment:
Doesn't this allow other functions outside of this struct implementation to change this value?

Reply:
Only if these functions have a mutable reference

Reply:
The renderer is mutabally borrowed in a several places. Probably best to keep it private.

Reply:
Done